includes/AMP/Sanitization.php

```diff
@@ -436,12 +435,14 @@ protected function get_sanitizers(): array {
 			$story = new Story();
 			$story->load_from_post( $post );
 
+			$poster_images = [
+				'poster-portrait-src' => esc_url_raw( $story->get_poster_portrait() ),
+			];
+
 			$sanitizers[ Story_Sanitizer::class ] = [
 				'publisher_logo' => $story->get_publisher_logo_url(),
 				'publisher'      => $story->get_publisher_name(),
-				'poster_images'  => [
-					'poster-portrait-src' => $story->get_poster_portrait(),
-				],
+				'poster_images'  => array_filter( $poster_images ),
```

Comment:
Little confused on the usage of array filter here. Doesn't this mean that an array is returned instead of string here?